public/css/common.less

```diff
@@ -303,8 +303,11 @@ form[name="form_confirm_removal"] {
 .affected-objects {
   display: inline-flex;
   align-items: baseline;
-  background-color: @state-critical;
-  color: @text-color-inverted;
+  color: @state-critical;
   padding: 0 0.25em;
   .rounded-corners();
+
+  .icon {
+    opacity: .9;
```

Comment:
Now the count is slightly darker than the icon, is that intention?

Reply:
Yes, since icons are usually bolder than fonts it's common practive to tone them down with opacity. We already do this in different places, e.g. menu nav items.